lifecycle_test.go

```diff
@@ -914,13 +914,10 @@ func TestPrintContainerLogsOnError(t *testing.T) {
 	rd := bufio.NewReader(containerLogs)
 	for {
 		line, err := rd.ReadString('\n')
-		if err != nil {
-			if err.Error() == "EOF" {
-				break
-			}
-
-			t.Fatal("Read Error:", err)
+		if err != nil && err.Error() == "EOF" {
```

Comment:
suggestion: use `errors.Is`
```suggestion
		if errors.Is(err, io.EOF) {
```